packages/auto_mappr/lib/src/builder/auto_mappr_builder.dart

```diff
@@ -17,6 +17,7 @@ class AutoMapprBuilder {
     'non_constant_identifier_names',
     'require_trailing_commas',
     'unnecessary_raw_strings',
+    'unnecessary_lambdas',
```

Comment:
This is due to the whenIsNullExpression being formatted as a function call e.g. `abc()` way before we use it in the enums firstWhere orElse part. It results in generating `() => abc()`. I felt like the problems I could introduce by trying to somehow fix this issue outweighed the drawbacks of having the unnecessary lambda.

Reply:
Hmm, I have fixed that for other defaults and customs I think. But it seems that because orElse takes a function, and whenSourceIsNull is either a function or a const, we would have to switch between doing a lambda returning const vs calling tear off, right?

Reply:
Exactly I looked at the other implementations but this one is a little special :D

Reply:
We will keep it this way as knowing if that is a const value or a function is a bit hard. 👍